src/components/app-progress-bar-line/AppProgressBarLine.js

```diff
@@ -0,0 +1,37 @@
+import { Typography } from '@mui/material'
+import { Box } from '@mui/system'
```

Comment:
Also, use imports like this, for all mui components: 
**import Button from '@mui/material/Button'**

Reply:
Fixed